src/overlays/actors/ovl_En_Hy/z_en_hy.h

```diff
@@ -12,25 +37,25 @@ typedef struct EnHy {
     /* 0x0000 */ Actor actor;
     /* 0x014C */ SkelAnime skelAnime;
     /* 0x0190 */ EnHyActionFunc actionFunc;
-    /* 0x0194 */ char unk_194;
-    /* 0x0195 */ u8 unk_195;
-    /* 0x0196 */ s8 unk_196;
-    /* 0x0197 */ s8 unk_197;
-    /* 0x0198 */ s8 unk_198;
-    /* 0x0199 */ s8 unk_199;
+    /* 0x0194 */ char unk_194; // unused
+    /* 0x0195 */ u8 pathReverse;
+    /* 0x0196 */ s8 objBankIndexHead;
+    /* 0x0197 */ s8 objBankIndexSkelUnused; // always the same as objBankIndexSkel, and doesn't achieve anything
```

Comment:
Mostly curious but did you try to investigate what happens if this isn't the same as `objBankIndexSkel`? It seems to be used for a specific purpose in `EnHy_PostLimbDraw`, even tho it's not relevant in the final game (or at least not in this version). Also not sure about calling it "unused" for this reason.

Reply:
I'm not sure what my reasoning was at the time. Maybe I thought that `SkelAnime_DrawFlexLimbOpa` was setting the segments back to their original values somehow, but it isn't

So the "doesn't achieve anything" part is only true due to the object being the same, but actually if it was to be different then half the skeleton (limbs 8-15 where 1 is root since the segment change is after limb 7) would use `objBankIndexSkelUnused` instead of `objBankIndexHead`

I'm confused what this was intended for. Maybe they planned to use half-skeletons of each hylian npc to boost the variety. This could make sense since it seems all hylian npcs used by EnHy use the same skeleton (eg limb 15 is the head limb for all of them)

<details>

<summary>
It sounded too fun to not try, right (click for details)
</summary>

```c
// at the top of EnHy_FindSkelAndHeadObjects
unusedSkelInfoIndex = Rand_ZeroOne() * 7.99;
```

Results (it's very prone to crashing, this was a lucky run):

![](https://421.es/doyu/28lb80)

![](https://421.es/doyu/28lb8z)

Vanilla for reference:

![](https://421.es/doyu/28lbby)

</details>

So yeah anyway I'm probably renaming (and in `EnHyModelInfo` too)
`objBankIndexSkel` -> `objBankIndexSkelFirstHalf`
`objBankIndexSkelUnused` -> `objBankIndexSkelSecondHalf`
?

Reply:
is it clear which part of the body each is for? upper and lower, for upper and lower body seems fine